docs/internals/howto-release-django.txt

```diff
@@ -52,22 +55,40 @@ You'll need a few things before getting started:
 * A GPG key. If the key you want to use is not your default signing key, you'll
   need to add ``-u you@example.com`` to every GPG signing command below, where
   ``you@example.com`` is the email address associated with the key you want to
-  use.
+  use. You will also need to add ``-i you@example.com`` to the ``twine``
+  shell invocation.
```

Comment:
`-u` is the correct option to use a non-default GPG key, and it's already documented. Please revert.